src/roaring64.c

```diff
@@ -456,7 +456,8 @@ bool roaring64_bitmap_contains_bulk(const roaring64_bitmap_t *r,
     uint8_t high48[ART_KEY_BYTES];
     uint16_t low16 = split_key(val, high48);
 
-    if (context->leaf == NULL || context->high_bytes != high48) {
```

Comment:
Ooo, yeah, this is bad, `array1 != array2` operates by decaying to pointers, so this would _never_ be true 😲. https://c.godbolt.org/z/7hh9nGTP8

Think we should write a test that fails without this change and add it..

Reply:
Ah, but if this is always false, it's just a performance impact, no correctness issue, so that's good. It just means we never get the benefit of using the context object to avoid having to lookup a leaf.